core/src/main/java/org/kohsuke/stapler/bind/BoundObjectTable.java

```diff
@@ -260,38 +254,22 @@ public Object getDynamic(String id) {
             return resolve(id);
         }
 
-        private synchronized Ref release(String id) {
-            return entries.remove(id);
+        /**
+         * Releases a particular id from the table.
+         * Normally would be called via {@link #releaseMe} which infers the id from the current request.
+         */
+        public void release(String id) {
```

Comment:
Making this public so we can clean up the reflection in https://github.com/jenkinsci/jenkins/commit/aac8c239721ed52caacbff389a29d5db2c1e3a00.